server/dist/src/main/resources/concord-server.conf

```diff
@@ -112,7 +112,9 @@ concord-server {
             # requires "CREATE EXTENSION" privileges
             createExtensionAvailable = "true"
 
-            secretStoreSalt = "${secretStore.secretStoreSalt}"
+            # plain values of secretstore.secretStoreSalt and secretstore.serverPassword. using these values for migration of secrets.
+            #secretStoreSalt = "q1"
```

Comment:
why not use `${secretStore.secretStoreSalt}` and `${secretstore.serverPassword}` as values? 
secretStoreSalt/serverPassword is required and should already be defined in the cfg, so user doesn't have to do anything.